community/kernel/src/main/java/org/neo4j/kernel/impl/api/LookupFilter.java

```diff
@@ -59,18 +59,18 @@ public static PrimitiveLongIterator exactIndexMatches( PropertyAccessor accessor
             return indexedNodeIds;
         }
 
-        IndexQuery[] numericPredicates =
+        IndexQuery[] filteredPredicates =
```

Comment:
We can't do any filtering for the geometric case here, since we don't have access to the actual value. Filtering is done on the cypher level (https://github.com/neo4j/neo4j/pull/10876/commits/8de5b4efb398332d2000beabe287ea5ce9902b61)

Reply:
Changed to filtering on kernel level. (https://github.com/neo4j/neo4j/pull/10702/commits/b89c467cb16bebaf3c0e70a433a1c12f8a883b4f)